operationalGuide.md

```diff
@@ -0,0 +1,174 @@
+
+## Provider and User of the API
+* Provider: All organizations which are able to implement API endpoints which meet the requirements of the Common API Card API specifications (e.g. provide mandatory fields) can act as provider. It is expected that most often Card Service Providers will act at least as technical providers of the Card API.
+* User: The Card API primarily targets Open Finance Use Cases described within each of the foreseen levels (i.e. level 1 to 3). The Card API is suitable for direct integration between Card Service Providers and third parties needing card data to enable new services or further develop existing use cases.
+
+Using the Card API for integrating Card Service Providers with distribution partners or issuers is not the main focus, as more detailed information and further enhanced use cases are usually required. 
+
+The contractual and technical requirements for exchanging card data and the allowed uses can be freely agreed upon between Provider and User. The standard does not set any specific rule in this regard. 
+
+
+# Central Design Decisions
+
+## Level concept
+The specification of the Card API is structured in different maturity levels, i.e. from an API modelling point of view, a Matryoshka approach is chosen here. This ensures that different users of the API with different availabilities of information still follow as similar an approach as possible. It also enables a more precise definition of the data model for data providers who do not want to offer all entities. Parties negotiate among themselves which level of the API should be used; ideally, level 3, the most comprehensive level of the API, is aimed for, as this ensures maximum compatibility.
+
+The levels build on each other. This means that level 1 supports the least functionality, but also requires the least information for use. Level 3 offers broader functionality with additional endpoints, but must also be used with complete data on the person/card/account. As the levels build on each other, it is also possible to communicate to lower levels via a proxy that filters the corresponding fields. For example, a data provider can have implemented level 3, but the proxy or end user can only map level 1 information and still receive corresponding responses for level 1.
+
+Please note that the three-tiered approach does not reflect the three card types, but different use cases as described below.
+
+![API Structure](https://github.com/swissfintechinnovations/ca-card/assets/116151702/f93b82c3-eba7-42db-9aac-f7e34037f936)
+
+## Name patterns for parameters
+
+### Differentiation between *code / *reference / *id
+The parameter names allow an easy recognition of the type of underlying identifier. The three following words are used as part of the parameter names for distinction:
+
+* *Id: technical identifier for an instance of a resource, is always in the uuid format; is used for direct access to a resource instance in an API endpoint
```

Comment:
- Is this ID standardised or dependent on each PSP?
- How does a third-party company obtain this technical information during onboarding, for example?
- What does the full user journey look like? A PAN is probably also required so that a third-party company initially obtains this data. Alternatively, access via IBAN (debit) or card account (credit) would be conceivable. Remember: the main User is a bank client, not a PSP.